app/scripts/app.js

```diff
@@ -192,11 +192,15 @@ angular.module('confRegistrationWebApp', ['ngRoute', 'ngCookies', 'ui.bootstrap'
                  * server side so the users access_token can be fetched to build the log out URL.
                  * after the GET, if successful, then delete the cookies. */
               } else if ($cookies.crsAuthProviderType === 'FACEBOOK') {
-                $http.get('auth/facebook/logout').success(function (data, status, headers) {
+                $http.get('auth/facebook/logout').success(function () {
                   delete $cookies.crsAuthProviderType;
                   delete $cookies.crsPreviousToken;
                   delete $cookies.crsToken;
-                  $window.location.href = headers('X-Facebook-Logout-URL');
+
+                  $facebook.logout().then(function(){
+                    $location.url('/#');
```

Comment:
If the path is local to the app, you can use $location.path('/'). But both work.